app/connectors/DesConnector.scala

```diff
@@ -63,48 +59,23 @@ trait DesConnector extends ServicesConfig with AuditService with RawResponseRead
     def read(http: String, url: String, res: HttpResponse) = customDESRead(http, url, res)
   }
 
-  def ctSubmission(ackRef:String, submission: JsObject, journeyId : String, isAdmin: Boolean = false)(implicit headerCarrier: HeaderCarrier): Future[DesResponse] = {
+  def ctSubmission(ackRef:String, submission: JsObject, journeyId : String, isAdmin: Boolean = false)(implicit headerCarrier: HeaderCarrier): Future[HttpResponse] = {
     val url: String = s"""${serviceURL}${baseURI}${ctRegistrationURI}"""
-    metricsService.processDataResponseWithMetrics[DesResponse](metricsService.desSubmissionCRTimer.time()) {
-      val response = cPOST(url, submission)
-      response flatMap { r =>
-        sendCTRegSubmissionEvent(buildCTRegSubmissionEvent(ctRegSubmissionFromJson(journeyId, r.json.as[JsObject])))
-        r.status match {
-          case OK =>
-            Logger.info(s"Successful Des submission for ackRef $ackRef to $url")
-            Future.successful(SuccessDesResponse(r.json.as[JsObject]))
-          case ACCEPTED =>
-            Logger.info(s"Accepted Des submission for ackRef $ackRef to $url")
-            Future.successful(SuccessDesResponse(r.json.as[JsObject]))
-          case CONFLICT =>
-            Logger.warn(s"ETMP reported a duplicate submission for ack ref $ackRef")
-            Future.successful(SuccessDesResponse(r.json.as[JsObject]))
-          case BAD_REQUEST =>
-            val message = (r.json \ "reason").as[String]
-            Logger.warn(s"ETMP reported an error with the request $message")
-            val event = new DesSubmissionEventFailure(journeyId, submission)
-            auditConnector.sendEvent(event) map {
-              _ => InvalidDesRequest(message)
-            }
+    metricsService.processDataResponseWithMetrics[HttpResponse](metricsService.desSubmissionCRTimer.time()) {
+      cPOST(url, submission) flatMap { response =>
+        sendCTRegSubmissionEvent(buildCTRegSubmissionEvent(ctRegSubmissionFromJson(journeyId, response.json.as[JsObject]))) map { _ =>
```

Comment:
If we receive an error response from the Audit call (as we saw recently) this will currently throw the "Submission to DES failed" log. This is one of 2 blocking Audit calls in the updateHeldSubmission function. _Should_ be fixed by a retry as part of the frontend work in this story [SCRS-5292]